src/vs/workbench/contrib/chat/browser/actions/chatContextActions.ts

```diff
@@ -237,23 +271,29 @@ class AttachFileToEditingSessionAction extends Action2 {
 			title: localize2('workbench.action.edits.attachFile.label', "Add File to {0}", 'Copilot Edits'),
```

Comment:
Won't this look really strange and be unusable in a lot of languages?

Reply:
Copilot Edits is a proper noun and shouldn't be localized, right?